tree-sitter-python/test/Main.hs

```diff
@@ -24,10 +24,12 @@ s `shouldParseInto` t = do
 
 pass = Py.PassStatementSimpleStatement (Py.PassStatement "pass")
 one = Py.ExpressionStatementSimpleStatement (Py.ExpressionStatement [Left (Py.PrimaryExpressionExpression (Py.IntegerPrimaryExpression (Py.Integer "1")))])
+function = Py.ExpressionStatementSimpleStatement (Py.ExpressionStatement [Left (Py.PrimaryExpressionExpression (Py.IdentifierPrimaryExpression (Py.Identifier "expensive")))])
```

Comment:
These broken tests also need to be transferred to a sep PR and remedied, accidentally committed because I'm sleepy.